astroquery/jplhorizons/core.py

```diff
@@ -584,6 +590,9 @@ def ephemerides_async(self, airmass_lessthan=99,
         else:
             request_payload['SKIP_DAYLT'] = 'NO'
 
+        if rts_flag:
+            request_payload['R_T_S_ONLY'] = rts_flag
+
```

Comment:
In addition it is helpful to check the parameter and provide an informative error message.

```python
if rts_flag and rts_flag not in ('TVH', 'GEO', 'RAD'):
    raise ValueError("Invalid rts_flag specified. Valid types are TVH, GEO and RAD")
```

Reply:
Not sure if it's a good idea if we hardcode current valid options.

As I understand this library just should be a "transparent" proxy of the end-service backend. A link to the service settings is provided in the docstring to be able to quickly identify the accepted options (to JPL docs).

I mean, if the backend changes (JPL in this case) a new PR should be raised to support new settings; if we keep this as simple as possible delegating the payloads validation to the backed we'll ensure that no downtime nor updates are needed to still support new functionalities.

It plays with the same idea that I propose in #1802, pass the payload to the backend delegating the validation and just raise if errors appears.

But if you think that this assertions should be performed I can change it without any effort :)

Thanks @migueldvb

Reply:
That sounds good to me. I think that his can be merged as it is if there are no conflicts with the master branch.

Reply:
Perfect!